package.json

```diff
@@ -33,40 +33,46 @@
     "redux-thunk"
   ],
   "dependencies": {
-    "express": "^4.16.3",
```

Comment:
no idea how and why these things got changed ...

Reply:
just checked all of these, they and their versions are all the same. Only vitessce is changed from `"^0.1.8"` to `"^0.2.1"` and this change was needed in order to solve the cell coordinates bug